### DISCUSSION TITLE
Jupiter

### DISCUSSION THREAD

**Discussion by giscus** [fedora-riscv/fedorav-force-images-discussions#4] [Q&A]
# Jupiter

https://images.fedoravforce.com/Jupiter

**Comment by tekkamanninja** [comment]
u-boot in SPI flash has is bug in env :
bootcmd=run detect_dtb; sysboot mmc ${boot_devnum}:${bootfs_part} any ${kernel_addr_r} /extlinux/extlinux.conf
the "mmc" should be  ${bootfs_devname}, to boot automaticly from NVME or SD card.

**Reply by Turixen** [reply]
in addition to this in my case to save this configuration it was also necessary to update a file named `env_k1-x.txt` located into `/boot`.

**Reply by frankenstein91** [reply]
do you know how to change that?

**Reply by Turixen** [reply]
Yes, sorry for latency, in my case I was serial connected to the board, and the prompt showed me the U-boot where is possible to modify the uboot env varibles.
After that when the OS is booted you have to go to /boot and change the env_k1-x.txt file

**Comment by OctopusET** [comment]
MD5 checksum is not matching for GNOME image, it's 236c40b0a4f4639b6e73d2d9a9057fd5

**Reply by imguoguo** [reply]
Thanks for the feedback, information has been updated

**Comment by OctopusET** [comment]
I tried `dd` on my NVMe drive, but it didn't boot. How can I use this image with NVMe drive?

**Reply by imguoguo** [reply]
Thanks for the feedback. The solution to the problem has already been demonstrated above. If you can connect to the console, perhaps you could try the solution mentioned earlier?

https://github.com/fedora-riscv/fedorav-force-images-discussions/discussions/4#discussioncomment-11168717

**Reply by OctopusET** [reply]
I tried @tekkamanninja 's solution, but it didn't work

**Reply by OctopusET** [reply]
I modified `env_k1-x.txt` file in `/dev/sda5` like this
```bootcmd=run detect_dtb; sysboot ${bootfs_devname} ${boot_devnum}:${bootfs_part} any ${kernel_addr_r} /extlinux/extlinux.conf```

I even tried remove following whitespace and change it to `${boot_devname}` but no success

**Comment by ionutcatana** [comment]
would it be possible to provide building instructions for the sd card image? I am trying to compile Fedora and Debian myself and it would be of great help.

**Reply by ionutcatana** [reply]
I tried this image and it doesnt have any GPU drivers (it's using llvmpipe). After a dnf upgrade the board doesn't boot anymore.

**Comment by zirangua** [comment]
Any one has the issue with the GPU drivers...
All works perfectly but the 3D acceleration does not work

**Comment by zirangua** [comment]
By they the xfce version is not booting... after boot drop into repair console... with option to continue or press Ctrl +D for maintenance
